src/twilio_api_util.ts

```diff
@@ -29,6 +31,31 @@ export async function sendMessage(
     databaseMessageEntry.twilioSendTimestamp = new Date();
   }
 
+  if (
+    options.deduplicationId &&
+    !(await deduplicate(options.deduplicationId))
+  ) {
+    logger.warn(
+      `TWILIOAPIUTIL.sendMessage: Not sending duplicate Twilio message to ${options.userPhoneNumber}: ${message}`
```

Comment:
Can we add more info in this log (e.g. Slack thread and Slack message ts, if available from the databaseMessageEntry)?